charts/meteor-pipelines/templates/validate-jupyterhub-image.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: validate-jupyterhub-image
+spec:
+  params:
+   - name: url
+  results:
+   - name: python-runtime
+     description: Python runtime
+   - name: pip-packages
+     description: Packages installed in runtime
+   - name: message
+     description: Messages passed down from task
+  steps:
+    - name: jupyter-book-build
+      image: $(params.url)
+      securityContext:
+        privileged: true
+      script: |
+         #!/usr/bin/env bash
+
+         # Default values for easier handling when error occurs
+         echo -n "" > $(results.message.path)
+         echo -n "[]" > $(results.python-runtime.path)
+         echo -n "[]" > $(results.pip-packages.path)
+
+         # Check Python version and save it to variable
+         echo  "Checking Python version"
+         version=$(python --version)
+         # Check if Python was found, if not fail task
+         if [[ $? != 0 ]]; then
+              echo "Python is not installed" >&2
+              echo "Python unavailable" > $(results.message.path)
+              exit 1
+         fi
+
+         # Create JSON from Python version
+         PYTHON_VERSION=$(echo "[{\"name\":\"Python\",\"version\":\"$( echo $version | cut -d ' ' -f 2)\"}]")
+
+         # Save existing Python packages
+         echo "Checking for existing Python packages"
+         PACKAGES=$(pip list --disable-pip-version-check --format json)
```

Comment:
I'm not sure we want to list all the packages here.. is there a way to recognize which packages are explicitly installed and which are dependency packages?